learn/wavemaker-release-notes/v11-2-5.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "WaveMaker 11.2.5 - Release date: 20 March 2023"
+id: "v11-2-5"
+sidebar_label: "v11.2.5 - latest"
+---
+This release includes bug fixes for accessiblity in widgets and In-memory sessions.
+
+---
+
+## Bug fixes
+
+- Fixed accessibility issues in data table, list, form, switch, tab, radio button, pop-over, carousel, calendar, date, checkbox, label and Nav bar widgets.
```

Comment:
@bhargavigundaa, can you help provide some details about what accessibility issues were fixed, maybe one example would help (if we can specify).